pages/dkp/kommander/2.2/DKP-insight/licensing/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Insight licensing
+title: Insight licensing
+excerpt: Insight licensing
+menuWeight: 20
+beta: true
+enterprise: false
+---
+
+For this initial release, DKP Insight is provided by default with no other action or purchase required on your part. However, future releases will require an add-on trial license or Enterprise license.
```

Comment:
I think we should be very careful about what we say about 'future releases' because they have a habit of changing before they ship...   I prefer weasel words like 'may' to 'will'.   Also, we may decide to package this differently.   I suggest we paint with a broad brush here,  I don't see any benefit in being specific about exactly what licenses might be required in the future.

Also, can I disable Insights if I can't afford the resources for it or don't find it useful?

```suggestion
For this initial release, DKP Insights is provided by default with no other action or purchase required on your part. However, future releases may require additional licenses.
```

Reply:
@kanna823 - Are we okay to remove this for now? I know you said we're definitely ONLY offering Tech Preview for 2.2 and then the plan is to have customers purchase a license moving forward, but is that absolutely for sure in future versions? Craig is right here, we're better off not saying anything if we're not 100% certain what's coming in next releases.

Reply:
yes, we can remove it.

Reply:
disabling insights in a k8s cluster will be added in DKP 2.3 release